docs/confluence/perplexed.md

```diff
@@ -43,12 +43,16 @@ SMT is the hardest nut to crack! There could be so many reasons that they occur.
 Those would show up if the code is inherently complex. Red flags to look for:
 
 1.  **Tool analyses failing.** Open the `statsdata.json` file. For example, if you have a link  
-    https://vaas-stg.certora.com/output/43260/dafd64136c1d71ba882c/?anonymousKey=a8e7a2b10c4f9f74ea55624c166d7edee8f1e2bc  
+    ```
+    https://vaas-stg.certora.com/output/43260/dafd64136c1d71ba882c/?anonymousKey=a8e7a2b10c4f9f74ea55624c166d7edee8f1e2bc
+    ```
     then the file would be in  
+    ```
     https://vaas-stg.certora.com/output/43260/dafd64136c1d71ba882c/**statsdata.json**?anonymousKey=a8e7a2b10c4f9f74ea55624c166d7edee8f1e2bc  
+    ```
     Look at the `ANALYSIS` key. If you see any `false` entries on keys that are not `constructor()` or `ecrecover` - report in #bugs.
     
-2.  **A non-empty “Call Resolution” warnings table.** Delegatecalls could be tricky and invalidate some of the Prover’s optimizations that were built to avoid timeouts. If such appear, try to scope out the verification to ignore the delegatecalls. Check with a security person, a more experienced user, or even the customer if you’re not sure if it’s safe to ignore.
+2.  **A non-empty “Call Resolution” warnings table.** Delegate calls could be tricky and invalidate some of the Prover’s optimizations that were built to avoid timeouts. If such appear, try to scope out the verification to ignore the `delegatecall`s. Check with a security person, a more experienced user, or even the customer if you’re not sure if it’s safe to ignore.
```

Comment:
The first "Delegate calls" here now has a space. The second "`delegatecall`s does not have a space.

It seems that the first is written with the English syntax (the space is needed) while the second is written as a technical keyword (so the space should be omitted). Is this correct?

Reply:
yes